addons/hr_recruitment/models/ir_ui_menu.py

```diff
@@ -10,11 +9,14 @@ class IrUiMenu(models.Model):
     def _load_menus_blacklist(self):
         res = super()._load_menus_blacklist()
         is_interviewer = self.env.user.has_group('hr_recruitment.group_hr_recruitment_interviewer')
-        is_user = self.env.user.has_group('hr_recruitment.group_hr_recruitment_user')
-        if not is_interviewer:
-            res.append(self.env.ref('hr.menu_view_hr_job').id)
-        elif is_interviewer and not is_user:
-            res.append(self.env.ref('hr_recruitment.menu_hr_job_position').id)
-        else:
-            res.append(self.env.ref('hr_recruitment.menu_hr_job_position_interviewer').id)
+        if not is_interviewer and (job_menu := self.env.ref('hr.menu_hr_job', raise_if_not_found=False)):
+            res.append(job_menu.id)
+        elif (
+            is_interviewer
+            and not self.env.user.has_group('hr_recruitment.group_hr_recruitment_user')
+            and (pos_menu := self.env.ref('hr_recruitment.menu_hr_job_position', raise_if_not_found=False)
```

Comment:
```suggestion
            and (pos_menu := self.env.ref('hr_recruitment.menu_hr_job_position', raise_if_not_found=False))
```
Sorry, I probably miss one parenthesis in my suggestion

Reply:
fixed already